ogr/abstract.py

```diff
@@ -220,6 +220,7 @@ def __init__(
         created_at: str,
         tarball_url: str,
         git_tag: GitTag,
+        project=None,
```

Comment:
```suggestion
        project: Optional["GitProject"]=None,
```

Reply:
```suggestion
        project: Optional["GitProject"],
```

Maybe, we can make it compulsory. What do you think?